lib/themes/dosomething/paraneue_dosomething/templates/system/patterns/gallery.tpl.php

```diff
@@ -9,11 +9,12 @@
  * - $items: An array of items to display. Each $item in $items contains:
  *   - $item['class']: The class to apply to the li (string).
  *   - $item['content']: Themed item content (string).
+ * - $roles: Data roles to be applied (string).
  *
  * @see paraneue_dosomething_get_gallery()
  */
 ?>
-<ul class="gallery -<?php print $layout; ?> <?php print $classes; ?>">
+<ul class="gallery -<?php print $layout; ?> <?php print $classes; ?>" data-role="<?php print $roles; ?>">
```

Comment:
@deadlybutter so this all looks good. Just a note with data roles... you typically name the "role" part something, and it's 1 value per role. So in the above, you shouldn't really add a list of roles to a data-role. For this scenario it could be something like `data-show-more="true"` or something like that... And then with jquery you can grab the role and check it. Grab it with one of jquery's helpers, like `$('some-container').data('show-more');  // will retrieve 'true'`